ios/browser/web/brave_web_client.mm

```diff
@@ -90,6 +85,17 @@ bool WillHandleBraveURLRedirect(GURL* url, web::BrowserState* browser_state) {
   return CWVWebView.webInspectorEnabled;
 }
 
-void BraveWebClient::SetLegacyUserAgent(const std::string& user_agent) {
-  legacy_user_agent_ = user_agent;
+bool WillHandleBraveURLRedirect(GURL* url, web::BrowserState* browser_state) {
```

Comment:
can you move these in a separate PR so the diff is more clear?

Reply:
Fixed. It was because I did a rebase on master, and it had conflicts where functions rearranged during rebase. It's fixed now and should be much simpler.